### PR TITLE
Added more improvements

### PR DESCRIPTION
**Pull request by macagua**
- Updated package documentarion

- Added i18n support

- Added Spanish translation

### CODE REVIEW

**Comment by macagua** [comment]
@hvelarde please check out this pull request and merge if all is :+1: (ok)

cc @ericof

**Comment by macagua** [comment]
@hvelarde What is wrong with the plone 5 travis-ci task?, It can not pass the build task :( 

I would like that it be possible to merge this improvements :) :+1:

File: CHANGES.rst [hunk 1]
```diff
@@ -4,4 +4,19 @@ Changelog
 1.0a1 (unreleased)
 ------------------
 
+- Fixed errors for code analysis test of this package
```
Comment (hvelarde) [inline CHANGES.rst]: @macagua please do not clutter the changelog; add only stuff useful to integrators and developers; IMO this is not useful

File: CHANGES.rst [hunk 2]
```diff
@@ -4,4 +4,19 @@ Changelog
...
+  [macagua]
+
+- Pinned setuptools in buildout configuration for use latest version
```
Comment (hvelarde) [inline CHANGES.rst]: neither this one

File: CHANGES.rst [hunk 3]
```diff
@@ -4,4 +4,19 @@ Changelog
...
+  [macagua]
+
+- Updated information about this package.
+  [macagua]
+
+- Added Spanish translation.
```
Comment (hvelarde) [inline CHANGES.rst]: this one and the following can be merged into one

File: CHANGES.rst [hunk 4]
```diff
@@ -4,4 +4,19 @@ Changelog
 1.0a1 (unreleased)
 ------------------
 
+- Fixed errors for code analysis test of this package
+  [macagua]
+
+- Pinned setuptools in buildout configuration for use latest version
+  [macagua]
+
+- Updated information about this package.
```
Comment (hvelarde) [inline CHANGES.rst]: what information? metadata?

File: README.rst [hunk 5]
```diff
@@ -1,15 +1,168 @@
...
+  :height: 416px
+  :width: 545px
+  :alt: The Plone Open Data API.
+
+  The Plone Open Data API.
+
+Features
+--------
+
+This package have by default two (02) open data pluggable as the
```
Comment (hvelarde) [inline README.rst]: as per best practices, do not break sentences; documentation is not code

File: README.rst [hunk 6]
```diff
@@ -1,15 +1,168 @@
...
+- Issue Tracker: http://github.com/plonegovbr/collective.opendata/issues
+
+- Source Code: http://github.com/plonegovbr/collective.opendata
+
+- Website: http://plone.org.br
+
+Support
+-------
+
+If you are having issues, please let us know sending a email to gov@plone.org.br.
```
Comment (hvelarde) [inline README.rst]: who said so? if someone sends an email there you have 99% chances it will be ignored

File: README.rst [hunk 7]
```diff
@@ -1,15 +1,168 @@
...
+
+License
+-------
+
+The project is licensed under the GNU General Public License v2 (GPLv2).
```
Comment (hvelarde) [inline README.rst]: this is in the package metadata

File: src/collective/opendata/i18n.sh [hunk 8]
```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+DOMAIN='collective.opendata'
```
Comment (hvelarde) [inline src/collective/opendata/i18n.sh]: we use the script defined here: https://github.com/collective/buildout.plonetest/blob/master/qa.cfg#L56:L84

File: src/collective/opendata/plugins/content/__init__.py [hunk 9]
```diff
@@ -27,10 +28,10 @@ class Content(DataPlugin):
     implements(IDataPlugin)
 
     name = 'content'
-    title = 'Content Metadata'
-    description = '''Content information'''
+    title = _(u'Content Metadata')
+    description = _(u'''Content information''')
 
-    portal_types = ['File', 'Document', 'Image', 'Event', 'News Item', 'Folder']
+    portal_types = [_(u'File'), _(u'Document'), _(u'Image'), _(u'Event'), _(u'News Item'), _(u'Folder')]
```
Comment (hvelarde) [inline src/collective/opendata/plugins/content/__init__.py]: those names are already translated in plone.app.locales, isn't it? thy don't just use that message factory?

File: src/collective/opendata/utils.py [hunk 10]
```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
+from DateTime import DateTime
```
Comment (hvelarde) [inline src/collective/opendata/utils.py]: don't change the sort order; we must follow plone.api conventions

File: README.rst [hunk 11]
```diff
@@ -1,15 +1,168 @@
...
+      "uri": "http://localhost:8080/Plone/apidata/content/Folder/761536d101414a47bc0e5494f51d97f1"
+    }
+  ]
+
+Translations
+------------
+
+This product has been translated into 
+
+- Spanish (thanks, Leonardo J. Caballero G.)
```
Comment (hvelarde) [inline README.rst]: you already stated that in the changelog

**Comment by hvelarde** [comment]
@macagua tests must pass first and you have to contact @jeanferri for a merge of this

File: README.rst (hunk 11 above)
Comment (macagua) [inline README.rst]: I am using this styleguide for Add ons following  http://docs.plone.org/about/documentation_styleguide_addons.html#readme-example

File: README.rst (hunk 6 above)
Comment (macagua) [inline README.rst]: Following the Doc Plone styleguide for Add ons, I added this from author_email in setup.py file, but I can erase this section ;-)

File: src/collective/opendata/i18n.sh (hunk 8 above)
Comment (macagua) [inline src/collective/opendata/i18n.sh]: ok I delete this script

File: src/collective/opendata/utils.py (hunk 10 above)
Comment (macagua) [inline src/collective/opendata/utils.py]: @hvelarde I changed the sort order to pass the code analysis test ;-)

Comment (hvelarde) [inline src/collective/opendata/utils.py]: FYI: https://github.com/plone/plone.api/issues/187

File: CHANGES.rst (hunk 1 above)
Comment (macagua) [inline CHANGES.rst]: ok done!

File: CHANGES.rst (hunk 2 above)
Comment (macagua) [inline CHANGES.rst]: ok done!

File: CHANGES.rst (hunk 4 above)
Comment (macagua) [inline CHANGES.rst]: README information like this https://github.com/Covantec/collective.opendata/commit/637f1dc95acb7f5819ca3e525c2751eff62f87a5

File: src/collective/opendata/plugins/content/__init__.py (hunk 9 above)
Comment (macagua) [inline src/collective/opendata/plugins/content/__init__.py]: @hvelarde good idea and done in the code. thanks :+1:

**Comment by macagua** [comment]
@jeanferri please can check this build on travis https://travis-ci.org/plonegovbr/collective.opendata/jobs/70648696 

It build job said "No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself. The build has been terminated"

**Comment by macagua** [comment]
@jeanferri thanks very you!!!

**Pull request merged**